filters/openpolicyagent/internal/envoy/policytype.go

```diff
@@ -0,0 +1,21 @@
+package envoy
+
+import "errors"
+
+type PolicyType string
+
+const (
+	IngressPolicyType PolicyType = "ingress"
+
+	EgressPolicyType PolicyType = "egress"
+
+	AppPolicyType PolicyType = "app"
```

Comment:
enum you do normally like this:
```
type PolicyType int

const (
	Ingress PolicyType = iota + 1
	Egress
        App
)
```

Reply:
Agreed. This is however part of how OPA integrates with Envoy. I.e. these exact strings are passed and also respected / required by Styra.

Let me check if we can remove these strings alltogether, I think they are just passed through, so we could also pass them as command line arg.

Reply:
I was able to remove this entire path and make it Skipper external config